Makefile

```diff
@@ -1,9 +1,9 @@
-HOSTNAME=kentik.com
-NAMESPACE=automation
-NAME=kentik-synthetics
-BINARY=terraform-provider-${NAME}
-VERSION=0.1.0
-OS_ARCH=linux_amd64
+HOSTNAME = kentik
+NAMESPACE = automation
+NAME = kentik-synthetics
+BINARY = terraform-provider-${NAME}
+VERSION := 0.1.0
```

Comment:
This seems to work fine for the kentik-cloudexports provider. 
```
VERSION := $(shell echo `git tag --list 'v*' | tail -1 | cut -d v -f 2`)
```
I do realize that we do not have any tag in this repo yet, so we will need to have a default (say 0.0.1) that will be used when the shell command does not produce anything.

Reply:
Created a task: https://clkentik.atlassian.net/browse/KNTK-293